src/components/Badge/Badge.tsx

```diff
@@ -1,10 +1,10 @@
-import React, {FunctionComponent} from 'react';
+import React from 'react';
```

Comment:
```suggestion
import type React from 'react';
```

You can also rewrite as `import type { FC } from 'react';`